docs/triaging.md

```diff
@@ -0,0 +1,90 @@
+# Triaging
+
+The term _triage_ originally comes from medicine and describes the process of quickly examining patients who are taken
+to a hospital in order to decide which ones are the most seriously ill and must be treated first.
+
+By _triaging issues_, we are evaluating problems that our customers are facing and provide the appropriate level of
+support. The goal is to provide attention to all open issues, to categorise them and alert people when there are issues
+of high severity. The goal is _not_ to fix all issues or answer all the questions coming from the open source community
+immediately.
+
+## Bug fixing 101
+
+Not every bug is equally critical or time sensitive. Some things reported as bugs aren’t even bugs. If you are unsure
+whether a things needs fixing, just reach out to your colleagues and get their opinion. When you do fix a bug, it should
+always go hand-in-hand with adding new tests (or improving existing ones), so we can avoid any regressions in the
+future.
+
+## Triaging workflow
+
+There are a few different ways to triage issues:
+
+1. You can look at the `#feed-web-sdk` channel in Slack. This channel will automatically receive a message every day in
+   the morning with issues that require triaging.
+2. You can look at the triage view in the Github Project Board: https://github.com/orgs/getsentry/projects/31/views/29
+3. (Also for external contributors) You can filter by `Waiting for: Product Owner` label:
```

Comment:
I think I would exclusively do point 3 here.

Reply:
This should be _especially_ for the team, and _also_ for external contributors. I want this to be the thing that folks joining the team read to know what to do, which is usually _not_ 3 here. Triaging is not that relevant for most external contributors (it may be sometimes, but way more important for us)